data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/special_damage_type.cfg

```diff
@@ -348,3 +348,272 @@
         {SUCCEED}
     [/event]
 )}
+
```

Comment:
Please:
* Put both of these tests in a separate file.
* Put both tests in the `resistance` subfolder.
* Use the same setup and format as the other tests of the resistance ability in the `resistance` subfolder.

Reply:
done

Reply:
Thanks for resolving this in eac1c92, but 7b0db4e unresolved it again.

Reply:
test fail with resistance format standart, eachtime alice receive 10pt of damage instead of 100 or 200

Reply:
Alice takes 110 damage, because the `-100%` resistance ability in eac1c92 is only active on defense. When attacking, Alice has elves' natural `-10%` to arcane.

Reply:
Iwill remove active_on  then